use crate::core::error::{PyForgeError, Result, validation};

pub fn run() -> Result<()> {
    println!("Building project");
    Ok(())
}